src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -114,7 +118,7 @@ public void houseReturnsExpectedItemWhenDismantled(
             house.tryDeliverItem(new Rock());
         }
         // Act
-        final IItem retrieveItem = house.tryDismantle(itemTypeToRetrieve);
+        final IItem retrieveItem = house.tryDismantle(itemTypeToRetrieve).orElse(null);
```

Comment:
You could use `.orElseThrow()` here to make the test fail if it can't dismantle the item.